src/ml-search.service.js

```diff
@@ -1128,6 +1121,37 @@
     },
 
     /**
+     * Retrieves values or tuples from 1-or-more lexicons
+     * @method MLSearchContext#values
+     *
+     * @param {String} name - the name of a `value-option` definition
+     * @param {Object} [params] - URL params
+     * @param {Object} [options] - search options, used in a combined query
+     * @return {Promise} a promise resolved with values
+     */
+    values: function values(name, params, options) {
+      // TODO: what other conditions
+      if ( !options && params && params.options) {
+        options = params;
+        params = null;
+      }
+
+      params = params || {};
+      params.start = params.start || 1;
+      params.limit = params.limit || 20;
```

Comment:
by default, `values()` calls are limited to 20 values. Overridable through the `params` argument.